fastapi_health/route.py

```diff
@@ -6,6 +6,8 @@
 from fastapi.responses import JSONResponse
 
 T = TypeVar("T")
+HealthcheckReturnT = Union[Dict[str, Any], bool]
+ConditionFuncT = Callable[..., Union[HealthcheckReturnT, Awaitable[HealthcheckReturnT]]]
```

Comment:
Why the "T"? 🤔

Reply:
It was mostly an abbreviation of Type to denote that it's `HealthcheckReturnType` to avoid any namespace conflicts.

It's only really a preference similar to how some people like to prefix interfaces with `I` in TypeScript, I'm more than happy to omit them :).

Reply:
yeah, can you remove it, please? 🙏

Reply:
Done :)